npm/ng-packs/packages/core/src/lib/abstracts/abstract.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface AbstractModel {
```

Comment:
I think we can rename it to OAuthResponse

Reply:
The user don't have to use Oauth, so I have Changed AbpAuthResponse